change/@internal-react-components-b7490699-acd9-4bb3-91a1-86788782aec9.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "Update Feature Toggle API",
```

Comment:
nit: for the changelog this doesn't explain what the change is as is likely won't be mentioned in the release notes - maybe `Allow custom menu items in ParticipantList`

Reply:
Thanks. I recently acquired a higher appreciation for better changelog after I did the release on Monday. I completely agree with this...